libs/gi-ui/src/App.scss

```diff
@@ -0,0 +1,16 @@
+/* For Rarity background*/
```

Comment:
Should we delete the frontend/app.scss and move it to use this? or is that already done if we use the gi-ui components? I just don't want to have duplicated css where if we change it, we need to change it in 2 places

Reply:
I don't think any of the frontend is using gi-ui component for this reason.
For frontend to use gi-ui, I think we'd need to amend app.scss, and also update `Theme` with the modular theme.

Reply:
... or just change implementation to use the new `content-` palettes